src/renderer/components/ft-input/ft-input.js

```diff
@@ -253,6 +253,10 @@ export default defineComponent({
         return
       }
 
+      if (event.getModifierState('Control') && event.key === 'u') {
```

Comment:
This does not work on MacOS

See https://github.com/FreeTubeApp/FreeTube/blob/v0.19.2-beta/src/renderer/App.js#L321-L327 for current implementation

Also it should detect both cases `u` and `U` in case of cap lock or something else

And also I think it should have 
```js
        // Early return
        return
```

Reply:
Can't test on a Mac but should probably work now, if you press Command+u inside the search box it should clear the text if there's any.